capstone/src/main/webapp/form-loader.js

```diff
@@ -32,7 +32,37 @@ function callScriptFunction() {
             }
         } else { 
             formData = JSON.stringify(result.response.result);
-            fetch("/form-handler", {method: "POST", body: formData});  
+            return fetch("/form-handler", {method: "POST", body: formData});
         }
+    }).then((resp) => {
+        if (resp.ok){
+            getForms();
+        } else {
+            alert("Error has occured");
+        }
+    });
+}
+
+function getForms() {
+    fetch("/form-handler").then(response => response.json()).then((formsList) => {
+        const formElement = document.getElementById("form-container");
+        formElement.innerHTML = "";
+        formsList.forEach((form) => {
```

Comment:
.forEach has some issues in JS, I recommend staying away from it and instead doing:

```js
for (form of formList) {
  formElement.appendChild(createTeacherFormElement(form.editURL));
  formElement.appendChild(createStudentFormElement(form.URL)); 
}
```